specification/cdn/resource-manager/Microsoft.Cdn/stable/2021-06-01/afdx.json

```diff
@@ -4628,28 +4631,17 @@
       ],
       "properties": {
         "secretSource": {
-          "description": "Resource reference to the KV secret",
+          "description": "Resource reference to the KV certificate. Expected to be in format of /subscriptions/{​​​​​​​​​subscriptionId}​​​​​​​​​/resourceGroups/{​​​​​​​​​resourceGroupName}​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​/providers/Microsoft.KeyVault/vaults/{vaultName}​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​/secrets/{certifcateName}​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​​",
```

Comment:
Typo

Reply:
Updated to Azure Key Vault and fixed typo.